cancermuts/datasources.py

```diff
@@ -729,7 +713,8 @@ def _parse_db_files(self, gene_id, genome_assembly_version = 'GRCh38',
                        cancer_site_subtype_1=None,
                        cancer_site_subtype_2=None,
                        cancer_site_subtype_3=None,
-                       metadata=[]):
+                       metadata=[],
+                       transcript_accession=None):
```

Comment:
can this argument come before the metadata section in the list of arguments? e.g.
```
self, gene_id, genome_assembly_version = 'GRCh38', transcript_accession=None, ...)
```

Reply:
also, should we consider making this a mandatory argument?

At this point I don't think it makes much sense to filter if we don't have it